Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "AbstractDifferentiation"
 uuid = "c29ec348-61ec-40c8-8164-b8c60e9d9f3d"
 authors = ["Mohamed Tarek <mohamed82008@gmail.com> and contributors"]
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
(how) is this a breaking change?

Reply:
The additional type parameter breaks uses of the `ForwardDiffBackend` type, e.g., `SomeOtherType{ForwardDiffBackend}` won't dispatch correctly anymore.